templates/bake/Template/index.twig

```diff
@@ -49,7 +49,7 @@
 {% endif %}
 {% if isKey is not same as(true) %}
 {% set columnData = Bake.columnData(field, schema) %}
-{% if columnData.type not in ['integer', 'float', 'decimal', 'biginteger', 'smallinteger', 'tinyinteger'] %}
+{% if columnData.type not in ['integer', 'float', 'decimal', 'biginteger', 'smallinteger', 'tinyinteger'] or columnData.null is same as(true)  %}
```

Comment:
Isn't the value the issue not the nullability?

Reply:
Yup, this probably isn't the correct fix. The value itself needs to be checked at "runtime".

So for nullable numeric columns the generated PHP code should be 
```php
<td><?= $entity->field === null ? '' : $this->Number->format($entity->field) ?></td>
```

Reply:
alright, then I misunderstood the problem. will adjust soon